src/feed-input.scss

```diff
@@ -0,0 +1,78 @@
+@import "./new-settings";
+@import "./mixins";
+
+$fd-feed-input-row-height: 3rem;
+$fd-feed-input-row-max-height: 20rem;
+$fd-feed-max-width: calc(100vw * 2 / 3);
+
+$block: #{$fd-namespace}-feed-input;
+$thumb: #{$block}__thumb;
+$textarea: #{$block}__textarea;
+$submit-btn: #{$block}__submit-button;
+
+.#{$block} {
+  @include fd-reset();
+
+  display: flex;
+  flex-direction: row;
+  padding: 1rem 0.5rem 1rem 1rem;
+  background-color: #fff;
+  background-color: var(--sapField_Background, #fff);
+
+  @include fd-screen(m) {
+    margin-top: 1rem;
+    margin-bottom: 1rem;
+  }
+
+  @include fd-screen(l) {
+    max-width: $fd-feed-max-width;
+  }
+
+  .#{$thumb} {
+    display: none;
+    flex: 1 0 auto;
+    width: $fd-feed-input-row-height;
+    height: $fd-feed-input-row-height;
+    margin: 0 0.5rem 0 0;
+    background-size: cover;
+    background-repeat: no-repeat;
+
+    @include fd-screen(m) {
+      display: block;
+    }
+  }
+
+  .#{$textarea} {
```

Comment:
`&__textarea`

Reply:
missing `fd-reset()` ?

Reply:
fd-reset is not needed as it's a modifier for `fd-textarea`